src/components/Template/Template.stories.tsx

```diff
@@ -174,313 +173,275 @@ const columnStylesComplex = [
   },
-  argTypes: {
-    aboveHeader: { table: { disable: true } },
-    breakout: { table: { disable: true } },
-    contentBottom: { table: { disable: true } },
-    contentId: { table: { disable: true } },
-    contentPrimary: { table: { disable: true } },
-    contentSidebar: { table: { disable: true } },
-    contentTop: { table: { disable: true } },
-    footer: { table: { disable: true } },
-    header: { table: { disable: true } },
-    sidebar: { description: "Only used in Storybook." },
-  },
   render: (args) => (
     <>
-      <SkipNavigation />
-      <Template>
+      <Template sidebar={args.sidebar}>
         <TemplateBreakout>
-          <TemplateAboveHeader>
-            <Placeholder variant="short">Above Header</Placeholder>
-          </TemplateAboveHeader>
-          <TemplateHeader>
-            <Placeholder variant="short">Header</Placeholder>
-            <Placeholder variant="short">Breadcrumbs</Placeholder>
-            <Placeholder>Hero</Placeholder>
-          </TemplateHeader>
+          <Breadcrumbs
+            breadcrumbsData={[
+              { url: "#", text: "Home" },
+              { url: "#", text: "Research" },
+              {
+                url: "#",
+                text: "Catalog",
+              },
+            ]}
+          />
+          <Hero
+            backgroundImageSrc={getPlaceholderImage()}
+            heroType="campaign"
+            heading={<Heading level="h1" id="1" text="Hero Campaign" />}
+            imageProps={{
+              alt: "Image example",
+              src: getPlaceholderImage("smaller"),
+            }}
+            isDarkBackgroundImage
+            subHeaderText={otherSubHeaderText}
+          />
         </TemplateBreakout>
-        <TemplateContent sidebar={args.sidebar}>
-          <TemplateContentTop>
-            <Placeholder variant="short">Content Top</Placeholder>
-          </TemplateContentTop>
-          {args.sidebar === "left" && (
-            <TemplateContentSidebar>
-              <Placeholder>Left Sidebar</Placeholder>
-            </TemplateContentSidebar>
-          )}
-          <TemplateContentPrimary>
-            <Placeholder>Main Content</Placeholder>
-            <Placeholder variant="short">More Content</Placeholder>
-          </TemplateContentPrimary>
-          {args.sidebar === "right" && (
-            <TemplateContentSidebar>
-              <Placeholder>Right Sidebar</Placeholder>
-            </TemplateContentSidebar>
-          )}
-          <TemplateContentBottom>
-            <Placeholder variant="short">Content Bottom</Placeholder>
-          </TemplateContentBottom>
-        </TemplateContent>
-        <TemplateFooter>
-          <Placeholder>Footer</Placeholder>
-        </TemplateFooter>
+        <TemplateTop>
+          <Banner
+            content="This is the top content area!"
+            heading="Content Top"
+            type="informative"
+          />
+        </TemplateTop>
+        <TemplateSidebar>
```

Comment:
You need to add a case for when "sidebar" is set to "none"

Reply:
Actually, this also _displays_ correctly when sidebar is either "left" or "right", but we do want to recommend that the DOM follows the styles as well. So if `sidebar="right"`, the DOM should be set in the same manner.